OptionSelector/DetailTableViewController.m

```diff
@@ -0,0 +1,100 @@
+//
+//  DetailTableViewController.m
+//  OptionSelector
+//
+//  Created by Umar on 8/14/15.
+//  Copyright © 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "DetailTableViewController.h"
+
+@interface DetailTableViewController ()
+
+@end
+
+@implementation DetailTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Uncomment the following line to preserve selection between presentations.
+    // self.clearsSelectionOnViewWillAppear = NO;
+    
+    // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+    // self.navigationItem.rightBarButtonItem = self.editButtonItem;
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+#warning Incomplete implementation, return the number of sections
+    return 1;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+#warning Incomplete implementation, return the number of rows
+    return self.category.type.count;
+}
+
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"Selected" forIndexPath:indexPath];
+    
+    NSString *typeChosen = self.category.type[indexPath.row];
+    cell.textLabel.text = typeChosen;
+    
+    
+    return cell;
+}
+
+
+/*
+// Override to support conditional editing of the table view.
```

Comment:
It's a good idea to remove these auto generated comments and lines (unless you actually need them).